src/daemons/GraphDaemon.cpp

```diff
@@ -133,6 +135,7 @@ int main(int argc, char *argv[]) {
     status = setupSignalHandler();
     if (!status.ok()) {
```

Comment:
`setupSignalHandler()` will always return `Status::OK()` 

Do you sure `nebula::WebService::stop()` will be called ?